app/src/services/subscriptionService.js

```diff
@@ -30,7 +31,9 @@ class SubscriptionService {
 
     static async createSubscription(data) {
         logger.info('Creating subscription with data ', data);
-        data.userId = data.loggedUser.id;
+        data.userId = data.loggedUser.id === 'microservice'
+            ? data.userId || data.loggedUser.id
```

Comment:
redundant and missing clarity ()

Reply:
Removed redundant `|| data.loggedUser.id`